fe/fe-core/src/main/java/com/starrocks/catalog/IcebergTable.java

```diff
@@ -170,6 +170,18 @@ public List<Column> getPartitionColumnsIncludeTransformed() {
         return allPartitionColumns;
     }
 
+    public PartitionField getPartitionField(String partitionColumnName) {
+        List<PartitionField> allPartitionFields = getNativeTable().spec().fields();
+        Schema schema = this.getNativeTable().schema();
+        for (PartitionField field : allPartitionFields) {
```

Comment:
Is there a issue with partition evolution?
spec 0:  source_id=2, fieldId=1000, name=ts_year, transform=VoidTransform
spec 1:   source_id=2, fieldId=1001, name=ts,   transform=Identity